client/src/app/queries/applications.ts

```diff
@@ -26,6 +26,7 @@ export const ApplicationsQueryKey = "applications";
 export const useFetchApplications = () => {
   const queryClient = useQueryClient();
   const { isLoading, error, refetch, data } = useQuery({
+    initialData: [],
```

Comment:
What's for ?

Reply:
@ibolton336 I'm also curious why this is necessary. Maybe we can merge this and follow up though, to unblock further work on the Issues page.

Reply:
@gildub this came up on slack, it was to prevent having to propagate a bunch of undefined checks everywhere the apps fetch is used. I talked with Ian and in my followup PR I plan to replace this with returning `data || []` from the hook to solve the same problem without changing the initial fetch behavior.